doc/sphinx/conf.py

```diff
@@ -200,10 +197,28 @@ def escape_splats(app, what, name, obj, options, lines):
     '"Courier New"', 'monospace'
 ]
 html_theme_options = {
-    'font_family': ','.join(font_families),
-    'head_font_family': ','.join(font_families),
-    'caption_font_family': ','.join(font_families),
-    'code_font_family': ','.join(code_font_families),
+    "show_toc_level": 2,
```

Comment:
We can probably delete the `font_families` lists, I think this is the only place they're used?